app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -93,24 +93,6 @@
   @include govuk-media-query($from: tablet) {
     border-width: $govuk-border-width;
   }
-
-  // this comes from the backend so we can't put a class on it
-  h2,
-  .gem-c-feedback__heading {
-    @include govuk-text-colour;
-    @include govuk-font(24, $weight: bold);
-    margin: 0;
-  }
-
-  p {
-    @include govuk-text-colour;
-    @include govuk-font(19);
-    margin: govuk-spacing(2) 0;
-  }
-
-  a {
-    @extend %govuk-link;
```

Comment:
we could replace this extend with 
    @include govuk-link-common;
    @include govuk-link-style-default;
    @include govuk-link-print-friendly;

Reply:
Cool. What are the equivalents for h2 and p?

Reply:
One question though - wouldn't this route add additional CSS weight? Whereas the approach I'm using (apply the relevant classes using JS) doesn't. I think I'd prefer to not add more CSS if possible.

Reply:
for heading-m
```
@include govuk-text-colour;
@include govuk-font($size: 24, $weight: bold);
```

Reply:
so we can then also remove the JS for adding classes (which would require at least a partial import of frontend anyway)

Reply:
`govuk-frontend` is going to be imported by whatever app is rendering the page with the feedback component on anyway.

Reply:
apps yes. I though the concern was for static to have to import frontend to use those classes

Reply:
With this change static shouldn't have to import anything from `govuk-frontend` other than the mixins, ie. no added CSS weight.

Reply:
unless i'm reading it incorrently, if you add `.govuk-heading-m`, etc class they won't be styled in static as it doesn't import those partials. by using mixin's you get the styles in static with a minor hit

Reply:
Ah - are you saying that's a page that's only styled by static? I was assuming all pages are rendered by an app as well as static. If that's the case, I'll go with your suggestion.

Related: for the second commit, is there an equivalent for inputs using the `@include` stuff you suggested above?

Reply:
judging by https://github.com/alphagov/static/blob/master/app/views/root/_error_page.html.erb 
hopefully someone with more knowledge can advise

as for the input, side from border and font-size, not much more
https://github.com/alphagov/govuk-frontend/blob/master/package/govuk/components/input/_input.scss#L10

Reply:
@kr8n3r I've updated as per your suggestion, and removed the JS changes. I've also checked the component on the 404 page, and made some specific overrides to make inputs appear correctly when it's just static rendering the page. We'll need to remove them at a later date, but fine for now.